morpho/processors/sampling/PyBindRooFitProcessor.py

```diff
@@ -7,22 +7,32 @@
 Date: 06/26/18
 '''
 
-try:
-    import ROOT
-except ImportError:
-    pass
+import ROOT
```

Comment:
The previous try and except catching is required for the online documentation. Can you check this doesn't break anything?
If not, it might be better to leave this as it was.

Reply:
Noah and I were trying to understand how this try/except clause makes sense – given that "ROOT" is used on the next line after the except clause. Can you please explain a bit more about why/how this is required or helpful?

Reply:
Since ROOT is not installed on the Read the docs server via the setup.py script, when this PyBindRootProcessor file gets imported, it will fail.
This try and except is our way to handle the case of the ROOT package not being present, especially, this ROOT package cannot be installed.
The document is available here: https://morpho.readthedocs.io/en/latest/

Is this clearer this way?
If there is a better method to do this without killing readthedocs or if readthedocs is no longer maintained, we can for sure implement this...

Reply:
I see, thanks. @nsoblath, do you know of any alternative approaches that might work, here?

Reply:
That makes me think that if you @taliaweiss have an account on RTD, I could add you as maintainer along with Noah and Ben (or whoever you think would be relevant)

Reply:
ROOT import: I'm going to look into using this, and I'll test on RTD: https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_mock_imports

RTD account: @taliaweiss, if it's all right with you, I'll give you access to Morpho on RTD.  I look at it so infrequently that I'll have to go remind myself how everything works.

Reply:
I just made an account on RTD. @nsoblath, thank you for planning to give me access to morpho on RTD - that would be great.